### ISSUE TITLE
Current Docker image broken when EJABBERD_SSLCERT_HOST not used

### ISSUE THREAD

**Issue by aspettl** [rroemhild/docker-ejabberd#158]
Hi

I received
```
Writing EJABBERD_SSLCERT_HOST to /opt/ejabberd/ssl/host.pem
/opt/ejabberd/scripts/pre/01_write_certifiates_from_env.sh: line 17: /opt/ejabberd/ssl/host.pem: Read-only file system
```
when I upgraded to the newest Docker image today.

This is caused by https://github.com/rroemhild/docker-ejabberd/commit/fd0ce988dd45f569c4f00e1784a3d9d802a367de because the test for existence of EJABBERD_SSLCERT_HOST evaluates to true always.

Should be simple to fix and, of course, I can also prepare a pull request if needed.

**Comment by rroemhild** [comment]
Thx! Fixed in 53e7d11.

**Comment by aspettl** [comment]
Thanks, that was quick!
